tests/test_coronagraphy.py

```diff
@@ -175,3 +175,33 @@ def test_lyot_coronagraph():
 	# Checks performance of the coronagraph and if the focal length does not introduce artifacts
 	assert (wf_foc.power.max() / norm) < 5e-3
 	np.testing.assert_allclose(wf_foc.power, wf_foc2.power)
+
+
+def test_knife_edge_coronagraph():
```

Comment:
This test passes, even when the `direction` parameter is completely ignored by the `KnifeEdgeLyotCoronagraph` class. We probably want to test that the E-field is the same but inverted for opposite-oriented knife edges.

Reply:
Added a test for both left/right and up/down symmetry.